org2tex

```diff
@@ -2,18 +2,15 @@
 
 set -e
 
-if [[ -z "${ORG_DIR}" ]]; then
-    ORG_DIR="$(find $HOME/.emacs.d/elpa -maxdepth 1 -type d -regex '.*org\(-plus-contrib\)*-[0-9]*')"
-    # -maxdepth *is* a GNUism, but it's been ported everywhere by now.
-fi
-
-if [[ -d "$ORG_DIR" ]]; then
-    emacs -Q --batch \
-        --eval "(add-to-list 'load-path \"$ORG_DIR\")" \
-        --eval "(require 'org)" \
-        --eval "(setq org-latex-listings 'minted)" \
-        --eval "(setq org-confirm-babel-evaluate nil)" \
-        --visit="$1" --funcall org-latex-export-to-latex
-else
-    echo "No org-mode directory found."
-fi
+emacs -Q --batch \
+	--eval "(require 'package)" \
+	--eval "(add-to-list 'package-archives '(\"org\" . \"https://orgmode.org/elpa/\"))" \
+	--eval "(package-initialize t)" \
+	--eval "(package-refresh-contents)" \
+	--eval "(package-install 'org-plus-contrib)"
```

Comment:
I understand why this is necessary but this is also potentially destructive for folks who already had org installed some other way :)

Consider, perhaps, moving this section into its own script that you only run for Travis?

Reply:
@lvh After the build is green, I'm open to merge this with you original code checking for ```org```.